### PR TITLE
Introduce QGaussWedge

### PR DESCRIPTION
**Pull request by peterrum**

### CODE REVIEW

File: include/deal.II/simplex/quadrature_lib.h [hunk 1]
```diff
@@ -50,6 +50,22 @@ namespace Simplex
...
+  template <int dim>
+  class QGaussWedge : public Quadrature<dim>
+  {
+  public:
+    /**
+     * Users specify a number `n_points_1D` as an indication of what polynomial
+     * degree to be integrated exactly.For details, see the comments of
+     * Simplex::QGauss.
+     */
+    explicit QGaussWedge(const unsigned int n_points);
```
Comment (drwells) [inline include/deal.II/simplex/quadrature_lib.h:66]:
```suggestion
    explicit QGaussWedge(const unsigned int n_points_1D);
```

File: include/deal.II/simplex/quadrature_lib.h [hunk 2]
```diff
@@ -50,6 +50,22 @@ namespace Simplex
...
+  /**
+   * Integration rule for wedge entities.
+   */
+  template <int dim>
+  class QGaussWedge : public Quadrature<dim>
+  {
+  public:
+    /**
+     * Users specify a number `n_points_1D` as an indication of what polynomial
+     * degree to be integrated exactly.For details, see the comments of
```
Comment (drwells) [inline include/deal.II/simplex/quadrature_lib.h:63]:
```suggestion
     * degree to be integrated exactly. For details, see the comments of
```

**Review by drwells — CHANGES_REQUESTED** [review]
Two small things

**Comment by peterrum** [comment]
@drwells Thanks for the review. I have made the changes.

**Review by drwells — APPROVED** [review]

**Pull request merged**